### DISCUSSION TITLE
VRT in Figma mobile & desktop libraries

### DISCUSSION THREAD

**Discussion by aweell** [Telefonica/mistica-design#1309] [Proposals]
### Goal

Add test scenarios for visual regression in the libraries that check all the visually hidden properties by default or different configurations of the components so is easier in the review merge of a branch to detect unexpected changes.

### Context

The current setup only allows the reviewers of a branch to detect changes in the default configuration of the components and sometimes is complicated to detect the entire impact of a change in a library.

### Impact

The test scenarios will allow to detect and solve problems in the libraries before merging to the main branch.

### Discussion Questions

* Should all the components with boolean properties have a test?
* Should all the tests live in the same page?

### Additional considerations

https://honzatmn.substack.com/p/visual-regression-testing-for-design

**Comment by yceballost** [comment]
We will include VRT in new tasks that include layers hidden. For old implementations, we are going to think about which cases are relevant to apply VRT.

#1335
